tests/test_requirements.py

```diff
@@ -0,0 +1,42 @@
+# Taken from Airlock:
+# https://github.com/opensafely-core/airlock/blob/ca23669bcf22bf719bd79103ecc53da0a7c53e2f/tests/test_requirements.py
+from pathlib import Path
+
+
+PROJECT_ROOT = Path(__file__).parents[1]
+
+
+def test_requirements_are_consistent():
+    # The production and development dependencies aren't intended to overlap (the dev
+    # dependencies are supposed to be _extra_ packages needed for development) but
+    # due to transitive dependencies the `requirements.dev.txt` file can end up
+    # including packages which are also included in `requirements.prod.txt`.
+    #
+    # We use a `--constraint` argument in `requirements.dev.in` to ensure that whenever
+    # we compile the development dependencies we stay consistent with the current
+    # production dependencies. However it's still possible, by doing things in a certain
+    # order, to accidentally end up with divergent versions. This can lead to confusion
+    # and makes the actual installed set of packages dependent on the order in which the
+    # files are read, which is bad.
+    #
+    # The below test ensures that wherever the two files specify the same dependency
```

Comment:
```suggestion
    # The below test verifies that wherever the two files specify the same dependency
```

I'm not sure if it makes sense to say a test that doesn't change anything "ensures" something. It can still be false after the test is run.

Reply:
I think your wording is more correct, but I'll leave this (and the other suggestion) because it's a duplication of the code taken from elsewhere.

Reply:
I'm not sure there's any motivation to keep this exactly like the original. But I don't feel strongly about it. It's easy enough to understand. It could be easier, that's all.